packages/plugin-seo/src/index.ts

```diff
@@ -176,6 +178,22 @@ const seo =
 
           return global
         }) || [],
+      i18n: {
+        ...config.i18n,
+        resources: {
+          ...config.i18n?.resources,
+          en: {
```

Comment:
I used deepMerge to this in core and will be necessary if we implement my other feedback.